templates/todo/api/csharp-cosmos-sql/Program.cs

```diff
@@ -26,5 +26,16 @@
     policy.AllowAnyMethod();
 });
 
+// Swagger UI
+app.UseSwaggerUI(options => {
+    options.SwaggerEndpoint("./openapi.yaml", "v1");
```

Comment:
For this to work, we need to have `openapi.yaml` placed into the `wwwroot` folder, and I suspect we need to modify the `repo.yaml` files for this project (and the mongo version) to ensure this file is copied to the right place:

https://github.com/Azure/azure-dev/blob/c414fc573d4785e6f9d875e3529ceb5ad0a2f974/templates/todo/projects/csharp-mongo/repo.yaml#L36-L40

Something around here needs to be updated.